src/main/resources/pom-servers.xml

```diff
@@ -579,8 +579,8 @@
             <id>kumuluzEE-1.2</id>
 
             <properties>
-                <kumuluz.version>2.5.2</kumuluz.version>
-                <kumuluzee-config-mp.version>1.1.1</kumuluzee-config-mp.version>
+                <kumuluz.version>3.2.0</kumuluz.version>
```

Comment:
We do not have the habit of updating the versions for older MicroProfile versions.

Reply:
Reverted to previous